### PR TITLE
Fixed a big bug.

### PR DESCRIPTION
**Pull request by TornaxO7**
I use arch btw

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -81,11 +81,12 @@
     wahrscheinlich auch ohne Ping schnell sehen.
 
 ## AUTHORS
-    /home/donald4444#3512, TornaxO7#7596, MultisampledNight#2425, 
+    /home/donald4444#3512, TornaxO7#7596, MultisampledNight#2425
```
Comment (MultisampledNight) [inline main.py:84]: very good

File: main.py [hunk 2]
```diff
@@ -95,7 +96,10 @@
     "Arch ist toll.",
     "I use Arch btw.",
     "AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAARCH!:wq",
-    "arch?"
+    "arch?",
+    "Habe ich Arch gehört?",
+    "Schonmal Arch installiert?",
+    "Arch for the win!"
```
Comment (MultisampledNight) [inline main.py:102]: incredible

Comment (TornaxO7) [reply main.py:102]: so doge

**Comment by MultisampledNight** [comment]
Looks good! Thanks!

**Pull request merged**